lib/local-links-manager/check_links/link_status_updater.rb

```diff
@@ -3,29 +3,27 @@
 module LocalLinksManager
   module CheckLinks
     class LinkStatusUpdater
```

Comment:
I've just looked at the HomepageStatusUpdater class for the first time. It looks like the updater was maybe planned to be abstracted into common library code at some point, but that hadn't happened yet. I'm not sure what the benefits of consistency are, but I agree that it will need a look at some point.

What with the plans to build a more encompassing Link Checker, do you agree that we should leave this code as-is, for now at least?